andela/kenya-rc
# #154715725 reviews and ratings

#### What does this PR do?
adds reviews and ratings functionality to products and shops

#### Description of Task to be completed?
- install react-stars to enable users rate a product
- create the productrating.js component
- create the productreview.js component
- add product rating and product review component to simple.js
- add shop reviews to subscriptions
- refactor product rating controller to match screen style
- add class names to product-review controller and remove inline styling
- add rate-shop button to tag.js file
- add shop-reviews register.js file
- add client index.js file for shop
- add component index.js file for shop
- add shop-rating controller
- add shop-review controller
- add template files for shop ratings and reviews

#### How should this be manually tested?
- navigate to the product page to add product reviews and ratings, the product shop is displayed 
  and rate-shop button is also displayed, clicking the button redirects to a shop review page.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#154715725

#### Screenshots (if appropriate)
<img width="1436" alt="screen shot 2018-02-06 at 5 19 56 pm" src="https://user-images.githubusercontent.com/32877365/35870878-0e770c20-0b63-11e8-9805-f8c2338c703c.png">

<img width="1432" alt="screen shot 2018-02-06 at 5 20 28 pm" src="https://user-images.githubusercontent.com/32877365/35870899-229d79aa-0b63-11e8-8632-2099e269c81f.png">

#### Questions:
N/A
